src_c/window.c

```diff
@@ -1066,6 +1066,35 @@ window_from_display_module(PyTypeObject *cls, PyObject *_null)
     return (PyObject *)self;
 }
 
+static PyObject *
+window_flash(pgWindowObject *self, PyObject *arg)
+{
+#if SDL_VERSION_ATLEAST(2, 0, 16)
+    long operation = PyLong_AsLong(arg);
+    if (operation == -1 && PyErr_Occurred()) {
+        return RAISE(PyExc_TypeError,
+                     "'operation' must be an integer. "
+                     "Must correspond with FLASH_CANCEL, FLASH_BRIEFLY, or "
+                     "FLASH_UNTIL_FOCUSED.");
+    }
+
+    if (operation != SDL_FLASH_CANCEL && operation != SDL_FLASH_BRIEFLY &&
+        operation != SDL_FLASH_UNTIL_FOCUSED) {
+        return RAISE(PyExc_ValueError, "Unsupported window flash operation.");
+    }
+
+    if (SDL_FlashWindow(self->_win, operation) < 0) {
+        return RAISE(pgExc_SDLError, SDL_GetError());
+    }
+#else
+    if (PyErr_WarnEx(PyExc_Warning, "'Window.flash' requires SDL 2.0.16+",
+                     1) == -1) {
+        return NULL;
+    }
```

Comment:
```suggestion
    return RAISE(pgExc_SDLError, "'Window.flash' requires SDL 2.0.16+");
```
I would prefer a `pygame.error` here instead

Reply:
The rest of the window API only sets warnings (so this would be incosistent), but maybe that's because they are attributes and not functions, should I raise the error anyways?

Reply:
@ankith26 I feel like raising `pygame.error` would mix up the "unsupported because platform" and "unsupported because low SDL" problems when catching it. I can still change it if needed.

Reply:
I feel like the mix up is fine because the user would want to handle both cases in the same way?

Reply:
Good point you got there. I'll ignore the inconsistency then